User/models.py

```diff
@@ -0,0 +1,144 @@
+from Core.models import PhonePrefix, WareHouse, Currency
+from django.core.validators import RegexValidator
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager, PermissionsMixin
+from django.db import models
+from django.dispatch import receiver
+from django.db.models.signals import post_save
+import re
+from django.core.exceptions import ValidationError
+from django.utils.text import slugify
+
+import random
+import string
+
+
+class UserManager(BaseUserManager):
```

Comment:
It would be better to put this to managers.py file, as in future if you have more, it will get messy